src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -46,13 +46,18 @@ import {
   ShippingAddressFormValues,
 } from "Apps/Order/Routes/Shipping2/shippingUtils"
 import { RouterLink } from "System/Router/RouterLink"
+import { log, trace } from "console"
```

Comment:
before I remove this: I learned that importing `log` from console can help when jest for some reason isn't logging your logs.